internal/config/config.go

```diff
@@ -158,6 +158,12 @@ type Clients struct {
 	InventoryBaseURL string `mapstructure:"inventory_base_url"`
 	// RbacBaseURL is the base endpoint to launch RBAC requests.
 	RbacBaseURL string `mapstructure:"rbac_base_url"`
+	// PendoBaseURL is the base url to reach out the pendo API.
+	PendoBaseURL string `mapstructure:"pendo_base_url"`
+	// PendoAPIKey indicates the shared key to communicate with the API.
+	PendoAPIKey string `mapstructure:"pendo_api_key"`
```

Comment:
I think we want to avoid serialising the API key, right?  In case it appears in a log output or something?
Rather, we should read it from an env var.